### PR TITLE
e2e tests

### PR DESCRIPTION
**Pull request by akellbl4**
- adds playwright as testing framework
- adds a test with comment posting as dev user

### CODE REVIEW

**Comment by akellbl4** [comment]
@umputun @paskal how can I pass `$BUILDPLATFORM` to [here](https://github.com/umputun/remark42/pull/1464/files#diff-194218c48b9a0cdd03974145733804c2d992ca818529fe2fa69a501d8b5b1cc3R24)
 https://github.com/umputun/remark42/runs/7965644294?check_suite_focus=true#step:4:9

**Review by Mavrin — DISMISSED** [review]

**Review by paskal — CHANGES_REQUESTED** [review]
Please tidy up the commit history. Twenty-eight commits, including merge ones, is too much.

Also, I expect something about e2e testing to be mentioned in the frontend Readme file.

File: compose-e2e-test.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    container_name: "remark42"
+
+    environment:
+      - REMARK_URL=http://remark42:8080
+      - SECRET=12345
+      - DEBUG=true
+      - ADMIN_PASSWD=password
+      - AUTH_DEV=true # activate local oauth "dev"
+      - ADMIN_SHARED_ID=dev_user # set admin flag for default user on local oauth2
+      - POSITIVE_SCORE=false # restricts comment's score to be only positive
```
Comment (paskal) [inline compose-e2e-test.yml:24]: False by default, no point adding it here.

File: compose-e2e-test.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - EDIT_TIME=5m # edit window
```
Comment (paskal) [inline compose-e2e-test.yml:25]: The default value as well should be removed.

File: compose-e2e-test.yml (hunk 1 above)
Comment (akellbl4) [reply compose-e2e-test.yml:24]: Then it should be fixed on dev composes.

File: compose-e2e-test.yml (hunk 2 above)
Comment (akellbl4) [reply compose-e2e-test.yml:25]: Same here

File: compose-e2e-test.yml (hunk 1 above)
Comment (paskal) [reply compose-e2e-test.yml:24]: Good point. Feel free to do so: it indeed doesn't make sense. I assume some frontend developer was playing with non-default values of these options and then didn't exclude them from the commit: 5809419bcece68350d55e4fe95076edfcf917250.

File: compose-e2e-test.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# compose for running e2e tests in CI
+version: "2"
+
+services:
+  remark42:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - BUILDPLATFORM=linux/amd64
```
Comment (paskal) [inline compose-e2e-test.yml:10]: Is that line needed?

**Review by paskal — APPROVED** [review]

**Review by Mavrin — APPROVED** [review]

**Pull request merged**